tests/unit/test_args_validator.py

```diff
@@ -164,6 +166,7 @@ def test_validate():
     assert SUT == argparse.Namespace(
         initial_cycle=-1,
         run_mode=1,
+        release_override=0,
```

Comment:
For the test this needs to be readded since release_override does not gets deleted in the `build_parser` function.